numpy/add_newdocs.py

```diff
@@ -663,14 +663,18 @@ def luf(lamdaexpr, *args, **kwargs):
         will only be made if __array__ returns a copy, if obj is a
         nested sequence, or if a copy is needed to satisfy any of the other
         requirements (`dtype`, `order`, etc.).
-    order : {'C', 'F', 'A'}, optional
+    order : {'C', 'F', 'A', 'K'}, optional, default 'K'
```

Comment:
Maybe better to use `no copy made` and `copy made`, as whether or not a copy is made will depend on other factors.